.github/workflows/cypress-tests.yml

```diff
@@ -37,22 +40,37 @@ jobs:
         with:
           ref: ${{ github.ref }}
 
-      - name: Setup node
-        uses: actions/setup-node@v4
-        with:
-          node-version: ${{ env.NODE_VERSION }}
-
-      - name: npm install
-        run: npm install
-
-      - name: Run cypress
-        run: npm run cy:run
+       name: Run
+        uses: cypress-io/github-action@v6
+	
         env:
-          url: ${{ secrets.AZURE_ENDPOINT }}
+          CYPRESS_url: ${{ secrets.URL }}
+        with:
+          browser: ${{ matrix.browser }}
```

Comment:
You can't use `matrix.browser` without the `strategy:` YML prose. Bring that over too